test/e2e/external_gateways.go

```diff
@@ -2950,8 +2998,17 @@ func checkAPBExternalRouteStatus(policyName string) {
 	gomega.Expect(status).To(gomega.Equal("Success"))
 }
 
-func createAPBExternalRouteCRWithStaticHop(policyName, namespaceName string, bfd bool, gateways ...string) {
+func createAPBExternalRouteCRWithStaticHopWithSuccessStatus(policyName, namespaceName string, bfd bool, gateways ...string) {
```

Comment:
I would leave the name as it was, since success status is the default/obvious option, and function behaviour didn't change (it created successful routes before)

Reply:
Sounds good. I've reverted the name change.